sdk/test-utils/recorder/ASSET_SYNC_WORKFLOW.md

```diff
@@ -87,34 +67,47 @@ npx dev-tool test-proxy push
 This command will:
 
 1. Push your local recordings to a tag in the `Azure/azure-sdk-assets` repo, and
-1. Update the `assets.json` in your package root to reference the newly created tag.
+2. Update the `assets.json` in your package root to reference the newly created tag.
+
+Example `assets.json` from "keyvault-certificates" SDK.
+
+```json
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "js",
+  "TagPrefix": "js/keyvault/keyvault-certificates",
+  "Tag": "js/keyvault/keyvault-certificates_43821e21b3"
+}
+```
+
+And the recordings are located at https://github.com/Azure/azure-sdk-assets/tree/js/keyvault/keyvault-certificates_43821e21b3
 
 You should stage and commit the `assets.json` update as part of your PR. If you don't run the `push` command before creating a PR, the CI (and anyone else who tries to run your recorded tests) will use the old recordings, which will cause failures.
 
 After onboarding your new package or after migrating your package to the asset-sync workflow, the following diagram describes the new workflow (new steps highlighted):
 
 ```mermaid
 graph TD
-    subgraph p3 [New steps]
-        subgraph p4 [Push recordings to asset sync repo]
+    subgraph p3 [Asset Sync]
+        subgraph p4 [Push to Assets Repo]
             push[npx dev-tool test-proxy push]
         end
-        p4-->assets[Commit <code>assets.json</code> change]
+        p4-->assets[Commit assets.json]
     end
 
-    assets --> pr[Push branch and\ncreate PR]
+    assets --> pr[Create PR]
 
-    subgraph p2 [Inspect recordings and test in playback]
-        playback[TEST_MODE=playback rushx test]
+    subgraph p2 [Verify in Playback]
+        playback[TEST_MODE=playback<br>rushx test]
```

Comment:
I think it would be more accurate to verify it works in the absence of .env file. From experience, it is easy for tests to accidentally depend on env vars in playback mode.

Reply:
As long as you set TEST_MODE to playback, we don’t care about the env variables in the recorder and instead pick the fake secrets. If that’s not happening, let’s sync up offline to see how to make the experience better.

Reply:
added anyway